root/etc/cont-init.d/50-config

```diff
@@ -208,7 +208,7 @@ if [ ! "$URL" = "$ORIGURL" ] || [ ! "$SUBDOMAINS" = "$ORIGSUBDOMAINS" ] || [ ! "
 fi
 
```

Comment:
Should `DUCKDNSTOKEN` also be printed out up on line 16?

When I ran into this issue last night, I assumed it wasn't there for security reasons, but, we're already writing the token to the file, so, that makes me think its probably not a security thing.  Might make debugging a problem easier for someone in the future.

Reply:
That part at the top is just so that there are some default values to compare to when trying to decide whether certs should be recreated. DUCKDNSTOKEN is not checked to see whether it changed so it's not needed to set it there.